### PR TITLE
[FR] Project Loved Team

### PR DESCRIPTION
**Pull request by Zekao**
Translation of  osu! wiki - Project Loved Team

### CODE REVIEW

File: wiki/People/The_Team/Project_Loved_Team/fr.md [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+- ![][flag_SG] [Abraxos](https://osu.ppy.sh/users/5025064)
+- ![][flag_PH] [lenpai](https://osu.ppy.sh/users/5314573)
+- ![][flag_GB] [Pope Gadget](https://osu.ppy.sh/users/2288341)
+
+## Trivia
+
+- ![][flag_US] [Toy](https://osu.ppy.sh/users/2757689) a crée la team originale des capitaines en Septembre 2017 avec le [Captain's Pick](/wiki/Beatmaps/History_of_Loved#captain's-pick-and-project-loved-(sep-2017-–-present)) systeme a été renommé plus tard en Project Loved. Fin 2019, il a surveillée et géré l'entièreté du projet.
+- Il y a un groupe d'utilisateurs Project Loved sur le site avec le group ID 31, mais la liste n'est pas publique. Elle est utilisée pour les autorisations de modéreration du [Project Loved forum](https://osu.ppy.sh/community/forums/120) et promu les beatmaps dans la catégorie Loved. Les seuls membres pouvant le faire sont ![][flag_US] [clayton](https://osu.ppy.sh/users/3666350) et ![][flag_US] [Toy](https://osu.ppy.sh/users/2757689).
+
+[flag_CL]: /wiki/shared/flag/CL.gif "Chile"
```
Comment (Pachiru) [inline wiki/People/The_Team/Project_Loved_Team/fr.md:59]:
```suggestion
[flag_CL]: /wiki/shared/flag/CL.gif "Chili"
```

File: wiki/People/The_Team/Project_Loved_Team/fr.md [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+[flag_CN]: /wiki/shared/flag/CN.gif "China"
```
Comment (Pachiru) [inline wiki/People/The_Team/Project_Loved_Team/fr.md:60]:
```suggestion
[flag_CN]: /wiki/shared/flag/CN.gif "Chine"
```

File: wiki/People/The_Team/Project_Loved_Team/fr.md [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+[flag_DE]: /wiki/shared/flag/DE.gif "Germany"
```
Comment (Pachiru) [inline wiki/People/The_Team/Project_Loved_Team/fr.md:61]:
```suggestion
[flag_DE]: /wiki/shared/flag/DE.gif "Allemagne"
```

File: wiki/People/The_Team/Project_Loved_Team/fr.md [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+[flag_GB]: /wiki/shared/flag/GB.gif "United Kingdom"
```
Comment (Pachiru) [inline wiki/People/The_Team/Project_Loved_Team/fr.md:62]:
```suggestion
[flag_GB]: /wiki/shared/flag/GB.gif "Royaume-Uni"
```

File: wiki/People/The_Team/Project_Loved_Team/fr.md [hunk 5]
```diff
@@ -0,0 +1,70 @@
...
+[flag_JP]: /wiki/shared/flag/JP.gif "Japan"
```
Comment (Pachiru) [inline wiki/People/The_Team/Project_Loved_Team/fr.md:63]:
```suggestion
[flag_JP]: /wiki/shared/flag/JP.gif "Japon"
```

File: wiki/People/The_Team/Project_Loved_Team/fr.md [hunk 6]
```diff
@@ -0,0 +1,70 @@
...
+[flag_LT]: /wiki/shared/flag/LT.gif "Lithuania"
```
Comment (Pachiru) [inline wiki/People/The_Team/Project_Loved_Team/fr.md:64]:
```suggestion
[flag_LT]: /wiki/shared/flag/LT.gif "Lithuanie"
```

File: wiki/People/The_Team/Project_Loved_Team/fr.md [hunk 7]
```diff
@@ -0,0 +1,70 @@
...
+[flag_LV]: /wiki/shared/flag/LV.gif "Latvia"
```
Comment (Pachiru) [inline wiki/People/The_Team/Project_Loved_Team/fr.md:65]:
```suggestion
[flag_LV]: /wiki/shared/flag/LV.gif "Lettonie"
```

File: wiki/People/The_Team/Project_Loved_Team/fr.md [hunk 8]
```diff
@@ -0,0 +1,70 @@
...
+[flag_NL]: /wiki/shared/flag/NL.gif "Netherlands"
```
Comment (Pachiru) [inline wiki/People/The_Team/Project_Loved_Team/fr.md:66]:
```suggestion
[flag_NL]: /wiki/shared/flag/NL.gif "Pays-Bas"
```

File: wiki/People/The_Team/Project_Loved_Team/fr.md [hunk 9]
```diff
@@ -0,0 +1,70 @@
...
+[flag_PH]: /wiki/shared/flag/PH.gif "Philippines"
+[flag_PL]: /wiki/shared/flag/PL.gif "Poland"
```
Comment (Pachiru) [inline wiki/People/The_Team/Project_Loved_Team/fr.md:68]:
```suggestion
[flag_PL]: /wiki/shared/flag/PL.gif "Pologne"
```

File: wiki/People/The_Team/Project_Loved_Team/fr.md [hunk 10]
```diff
@@ -0,0 +1,70 @@
...
+[flag_SG]: /wiki/shared/flag/SG.gif "Singapore"
```
Comment (Pachiru) [inline wiki/People/The_Team/Project_Loved_Team/fr.md:69]:
```suggestion
[flag_SG]: /wiki/shared/flag/SG.gif "Singapour"
```

File: wiki/People/The_Team/Project_Loved_Team/fr.md [hunk 11]
```diff
@@ -0,0 +1,70 @@
...
+[flag_US]: /wiki/shared/flag/US.gif "United States"
```
Comment (Pachiru) [inline wiki/People/The_Team/Project_Loved_Team/fr.md:70]:
```suggestion
[flag_US]: /wiki/shared/flag/US.gif "États-Unis"
```

**Review by Pachiru — CHANGES_REQUESTED** [review]
Apply this and it'll be good!

**Comment by DoryanR** [comment]
The translation needs to be revised, since changes have been made to the English version in the meantime.

**Review by DoryanR — CHANGES_REQUESTED** [review]
The translation needs to be revised, since changes have been made to the English version in the meantime.

**Comment by DoryanR** [comment]
@Zekao Are you still on the job?

**Comment by Zekao** [comment]
> @Zekao Are you still on the job?

I really don't have time to fix it for now so if you want to do a new pull you can ^^

**Comment by DoryanR** [comment]
This PR can be closed, following #5520

**Pull request closed without merging**